examples/experimental/i3s-17-and-debug/app-debug.js

```diff
@@ -315,7 +319,18 @@ export default class App extends PureComponent {
   }
 
   _renderDebugPanel() {
-    return <DebugPanel onOptionsChange={this._setDebugOptions}>{this._renderStats()}</DebugPanel>;
+    const {warnings, debugOptions} = this.state;
```

Comment:
Nit: Let's be consistent with underscores `_renderDebugPanel` vs `renderAttributesPanel`. Either no underscores or in all places.

Reply:
Renamed.